benchmarks/test_strategy.py

```
@@ -0,0 +1,102 @@
+)
+def test_strategy_detection(
+    benchmark: BenchmarkFixture,
+    version: str,
+) -> None:
+    """Benchmark for AwesomeVersion strategy detection."""
+
+    @benchmark
+    def _run_benchmark() -> None:
+        for _ in range(DEFAULT_RUNS):
+            obj = AwesomeVersion(version)
+            obj.strategy
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)